config/default.yml

```
@@ -423,6 +423,14 @@ Rails/EnumHash:
   Include:
     - app/models/**/*.rb
 
+Rails/EnumKeywordArgs:
+  Description: 'Use positional arguments over keyword arguments when defining enums.'
+  StyleGuide: 'https://rails.rubystyle.guide#enums'
+  Enabled: pending
+  VersionAdded: '<<next>>'
```

Can you add the `Severity: warning` setting:

```suggestion
  Enabled: pending
  Severity: warning
  VersionAdded: '<<next>>'
```